src/Algolia.Search/Models/Rules/ConsequenceParams.cs

```diff
@@ -34,9 +34,17 @@ namespace Algolia.Search.Models.Rules
     public class ConsequenceParams : Query
     {
         /// <summary>
-        /// When providing an object, it describes incremental edits to be made to the query string.
+        ///
         /// </summary>
-        /// <remarks>Setting a ConsequenceQuery will override SearchQuery if set. Both can't be set at the same time.</remarks>
+        public ConsequenceParams()
+        {
+        }
```

Comment:
Why is this constructor now needed?

Reply:
I needed to explicitly add this ctor because `System.Text.Json` was crashing otherwise. Maybe it has been fixed since then.